src/hooks/useAriaLive/useAriaLive.ts

```diff
@@ -0,0 +1,7 @@
+export const useAriaLive = (ariaLiveText: string): void => {
+  const ariaLiveElement = document.querySelector('#h5p-vocabulary-drill-aria-live');
```

Comment:
@henriettemoe Could you please use a `ref` instead of using a query selector (and feel free to make the aria-live region a component of its own if that makes sense) or at least a really unique id? As in my review before: You never know when multiple H5P instances of a content type will be present in the DOM (Vocabulary Drill may be used as subcontent) and with `h5p-vocabulary-drill-aria-live` only as id you'd have the same id in the DOM multiple times.

Reply:
We concur - these are good insights and we'll probably create a component here. The PR is marked as draft and with the WIP label and we'll keep working on it before another review is necessary, thanks!